runtime/src/main/kotlin/kotlin/text/Strings.kt

```diff
@@ -221,7 +221,7 @@ public actual fun CharSequence.repeat(n: Int): String {
 /**
  * Converts the characters in the specified array to a string.
  */
-public actual fun String(chars: CharArray): String = fromCharArray(chars, 0, chars.size)
+public actual fun String(chars: CharArray): String = unsafeStringFromCharArray(chars, 0, chars.size)
```

Comment:
`concatToString()` ?

Reply:
For some reason I thought it was a partial conversion, hence left `unsafeStringFromCharArray` to avoid bounds checking.

Reply:
done!